www/src/components/docsearch-content.js

```diff
@@ -4,7 +4,7 @@ export default ({ children }) => (
   // todo settle on one className to facilitate Algolia DocSearch;
   // `post` and `post-body` are only in use for the blog:
   // https://github.com/algolia/docsearch-configs/blob/6c2fd2d71302e3ead9a0a83dd44dca02ae0701e3/configs/gatsbyjs.json#L66-L77
-  <main id={`reach-skip-nav`} className={`docSearch-content post post-body`}>
+  <main id={`reach-skip-nav`} className={`docSearch-content post`}>
```

Comment:
Just so I'm aware, this is the json we're using now via Algolia right?

https://github.com/algolia/docsearch-configs/blob/6c2fd2d71302e3ead9a0a83dd44dca02ae0701e3/configs/gatsbyjs.json

Maybe we update the comment?

Reply:
Oh yeah, I can see how that's a bit misleading. Great catch! 🙏 

In #12464 I moved both `post` and `post-body` classnames to `docsearch-content` to make a first step to consolidate that stuff, write that comment, and keep track of stuff in one place.

When I was looking at the inline-CSS-in-docsearch-results issue today, I noticed that I had forgotten to actually remove `post-body` in `template-blog-post` — this is why I removed it in `docsearch-content` again in this PR.

Will push an update in a bit sorting this out!

I will give https://community.algolia.com/docsearch/config-file.html a good read tomorrow and add a PR over at https://github.com/algolia/docsearch-configs that finally resolves this: I want to have one selector (`.docSearch-content`) for that stuff (unless you stop me 😉).

Reply:
Updated in 4b831a9!